src/components/parameters.js

```diff
@@ -68,9 +68,12 @@ const useStyles = makeStyles((theme) => ({
     title: {
         padding: theme.spacing(2),
     },
+
     grid: {
-        padding: theme.spacing(2),
+        paddingTop: '16px',
```

Comment:
use theme spacing instead of hard coded values
here, theme.spacing(1) = 8px, theme.spacing(2) = 16px